pdns/recursordist/docs/lua-config/proxymapping.rst

```diff
@@ -48,7 +48,7 @@ See :func:`protobufServer` on how to tune the source address logged in ``Protobu
   Specify a table based mapping for a subnet.
 
   :param string subnet: a subnet to match
-  :param string ip: the IP address or IPaddress port combination to match the subnet to.
+  :param string ip: the IP address or IP address + port combination to match the subnet to.
```

Comment:
This was masked by the previous rule... The choice of `+` was arbitrary -- I'm definitely not attached to it.